Dockerfile

```diff
@@ -1 +1,38 @@
-FROM 345280441424.dkr.ecr.ap-south-1.amazonaws.com/base_centos:7-20210630
+FROM 345280441424.dkr.ecr.ap-south-1.amazonaws.com/ark_base_java8:latest
+
+LABEL ORG="Armedia LLC" \
+      VERSION="1.0" \
+      IMAGE_SOURCE="https://github.com/ArkCase/ark_solr" \
+      MAINTAINER="Armedia LLC"
+
+ENV SOLR_VERSION="7.7.2" \
+    SOLR_USER="solr" 
+    
+
+ENV SOLR_DOWNLOAD_URL https://archive.apache.org/dist/lucene/solr/${SOLR_VERSION}/solr-${SOLR_VERSION}.tgz
+ENV SOLR_PORT 8983
+
+ADD $SOLR_DOWNLOAD_URL /tmp
```

Comment:
I don't really see the point of the `SOLR_DOWNLOAD_URL` variable here... Why not expand it in the `ADD` command?

Reply:
@naveen-armedia  please make this change.

Reply:
@naveen-armedia @fabricetriboix  is saying why create SOLR_DOWNLOAD_URL, why not just make the add line:
`ADD "https://archive.apache.org/dist/lucene/solr/${SOLR_VERSION}/solr-${SOLR_VERSION}.tgz" /tmp`

Reply:
Yeah thanks @JonHolman updated with ADD command.

Reply:
Thanks @naveen-armedia .  I realize we should update to the get-artifacts.sh (pre build script) pattern at this time, so we don't rely on the internet download.

Reply:
@JonHolman Yeah updated with get-artifacts.sh